Library/Formula/youtube-dl-gui.rb

```diff
@@ -0,0 +1,53 @@
+class YoutubeDlGui < Formula
+  desc "A cross platform front-end GUI of the popular youtube-dl."
+  homepage "https://github.com/MrS0m30n3/youtube-dl-gui"
+  url "https://github.com/MrS0m30n3/youtube-dl-gui.git",
+    :revision => "95189d3d82c159e069697fa65058c8a9f88ff686"
```

Comment:
You should be able to use the tarball here. Upstream tagged one for you in https://github.com/MrS0m30n3/youtube-dl-gui/releases/tag/0.3.8.